### ISSUE TITLE
Official Soundtrack

### ISSUE THREAD

**Issue by ThomasGaubert** [JPII/NavalBattle#151]
This covers the implementation of music (excluding sound effects, as mentioned in #152).

Feel free to post suggestions in regards to how the music should work and of course the music itself. Please keep copyright in mind.

**Comment by smeagle42** [comment]
I need help with what y'all are looking for musically. Whether it be like dubstep, rock, metal, hardrock, soundeffects, whatever. If you want me to choose let me know.

**Comment by ThomasGaubert** [comment]
Sound effects would be #152, and @maximusvladimir  described where you can find some of them.

Otherwise, I'm not sure what type of genre we are looking for. IMO adventure music (Skyrim, etc.) might work, but I'm not 100% sure.

**Comment by maximusvladimir** [comment]
Whatever it is, it shouldn't be too intense, or with a fast beat. What about the SimCity 4 Deluxe music?

**Comment by ThomasGaubert** [comment]
Oh, yes, good idea.

I wonder if there are fair-use versions of the music.

**Comment by smeagle42** [comment]
We are not using someone else's created music! I am not going through the entire process of getting the copy right license so i'm creating our own. Yes i think we need slow music like in skyrim but we don't have the money to higher a 50 piece orchestra to do it so i'm going over to my brother's soon to make the music.

**Comment by maximusvladimir** [comment]
Freesound.org has ALOT of free, commercial, royalty-free music.

**Comment by ThomasGaubert** [comment]
> Freesound.org has ALOT of free, commercial, royalty-free music.

Yeah this would be our best best. Recording should be a last resort item, imo.

**Comment by ThomasGaubert** [comment]
Moved along with the sound effects issue.

**Comment by maximusvladimir** [comment]
Sounds good.
